h2o-core/src/test/java/water/fvec/NewChunkTest.java

```diff
@@ -1,622 +0,0 @@
-package water.fvec;
```

Comment:
Should this be deleted?

Reply:
I renamed it to NewChunkSpeedTest.java.  See above.

Reply:
Nope.  I need to add this back.  Sorry